DependencyInjection/Configuration.php

```diff
@@ -83,6 +83,8 @@ public function getConfigTreeBuilder()
                             ->defaultValue('auto')
                         ->end()
                         ->scalarNode('formatter')->defaultNull()->end()
+                        ->booleanNode('inject_debugger_for_discovered_http_client')->defaultTrue()->end()
```

Comment:
Can we find a shorter name? Like `profile_discovered_client` and `profile_discovered_async_client`